### PR TITLE
[MER-1121] Fix flaky community live tests

### PR DESCRIPTION
**Pull request by julmontesdeoca**
[MER-1121](https://eliterate.atlassian.net/browse/MER-1121)

What does it change?

- Fixes flaky community and admin unit tests due to sorting

### CODE REVIEW

File: test/oli_web/live/admin_live_test.exs [hunk 1]
```diff
@@ -259,19 +259,31 @@ defmodule OliWeb.AdminLiveTest do
     end
 
     test "applies sorting", %{conn: conn} do
-      _user_1 = insert(:user, %{given_name: "Testing A"})
-      user_2 = insert(:user, %{given_name: "Testing B"})
+      _user_a = insert(:user, %{given_name: "Testing A", email: "eeee@example.com"})
```
Comment (mcass19) [inline test/oli_web/live/admin_live_test.exs:262]: We could remove this line?

**Review by mcass19 — APPROVED** [review]
Thanks for fixing this 💪🏼 🚀

**Review by gastonabella — APPROVED** [review]
🥇

**Review by jrissler — APPROVED** [review]

File: test/oli_web/live/admin_live_test.exs (hunk 1 above)
Comment (julmontesdeoca) [reply test/oli_web/live/admin_live_test.exs]: Makes sense, updated 🙂

**Review by eliknebel — APPROVED** [review]

**Pull request merged**